acceptance-test/user-pathways/happy-path/happy-path.test.js

```diff
@@ -195,9 +198,21 @@ describe.only('User path(s)', () => {
         // supporting-documents-add
         await clickSelector(page, UPLOAD_DOCUMENT_PAGE_2_NO_OPTION);
         await clickSelector(page, CONTINUE_BUTTON);
-        // Run through the skeleton until we reach the end
+        // fr-details
+        await page.waitForSelector(FR_DETAILS_NAME_INPUT);
+        await page.$eval(FR_DETAILS_NAME_INPUT, (element) => {
+            element.value = 'Test';
```

Comment:
I'd put in a real name like 'Mo' or 'Ahmed'

Reply:
No worries I added them in another PR I'll copy them across

Reply:
Note: it doesn't need to be Mo or Ahmed 😄